api/src/main/java/io/strimzi/api/kafka/model/ZookeeperClusterSpec.java

```diff
@@ -180,6 +181,8 @@ public void setMetrics(Map<String, Object> metrics) {
     @Description("Pod affinity rules.")
```

Comment:
The description used to be the same on all these resources. Now you are changing it on half of them. Is there some reason for it?